docs/source/programming_resources/tutorial_specific/android_studio/fork_and_clone_github_repository/Fork-and-Clone-From-GitHub.rst

```diff
@@ -573,6 +566,12 @@ compiling against v7.2 use.
 
 The -Xtheirs option is a convenience that says, "If there are any conflicts, automatically take the software from the v8.0^2 side."
 
+   .. figure:: images/revert.png
+      :align: center
+      :alt: demonstrating the revert
+
+      Result of revert - a new merge commit representing the revert from v8.0 to v7.2.
+
 Summary
 -------
```

Comment:
Oops event_setup.rst wasn't supposed to be added to the commit.  You can now guess what I'd like my next page for ftc-docs to be.

Reply:
Sounds good to me.